src/pages/leeszaal/[readingRoomSlug]/[objectId]/index.tsx

```diff
@@ -63,10 +65,21 @@ const ObjectDetailPage: NextPage = () => {
 		router.query.objectId as string
 	);
 
-	// Set default view
+	const { data: playableUrl, isLoading: isLoadingPlayableUrl } = useGetMediaTicketInfo(
+		mediaUrl ?? '',
+		!!mediaUrl
+	);
```

Comment:
you can also handle that in the media service in the getPlayableUrl function. That way if any other code calls the mediaService.getPlayableUrl directly that will also have the benefit of this check:

```
	public static async getPlayableUrl(
		referenceId: string | null
	): Promise<string | null> {
	        if (!referenceId) {
		       return null;
	        }
		return await ApiService.getApi()
			.get(
				stringifyUrl({
					url: `${MEDIA_SERVICE_BASE_URL}/${MEDIA_SERVICE_TICKET_URL}`,
					query: {
						id: referenceId,
					},
				})
			)
			.text();
	}
```

Reply:
`!!mediaUrl` will prevent the mediaservice from being called until the actual url is available to the view, IMO the service should dictate its requirements, not fire empty promises while its waiting for actual input